halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/Telemetry.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 Armory, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.halyard.config.model.v1.node;
+
+import java.util.UUID;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode(callSuper = false)
+public class Telemetry extends Node {
+
+  @Override
+  public String getNodeName() {
+    return "telemetry";
+  }
+
+  private Boolean enabled = true;
```

Comment:
Are we sure that we want to enable this by default at the same time we're rolling this out to everyone?  Would this benefit from an opt-in period before moving to opt-out?

Reply:
You mean opt-*in* until 1.17, right?

Reply:
Just merge changes that make this opt-in for now.

Reply:
Yes, opt-in, sorry.